CMakeLists.txt

```diff
@@ -71,6 +71,8 @@ string(
 
 target_compile_features(cloysterhpc_options INTERFACE cxx_std_${CMAKE_CXX_STANDARD})
 
+add_compile_definitions(TEST_SAMPLE_DIR="${CMAKE_SOURCE_DIR}/test/sample")
```

Comment:
It will expand to `downloadPath/src/test/sample`? If yes I think we should cover the `CHROOT` [here](https://github.com/viniciusferrao/cloysterhpc/blob/b206a89e8570e2bbed04caf059d1b695830e844f/include/cloysterhpc/const.h#L25).

Reply:
Yes

But this is used (or at least _should_ be used) only during unit test execution

I don't think anyone with a sane mind would run them inside of a chroot... they are not meant for that.

Reply:
I'm still not convinced. I was only saying that maybe the `CHROOT` thing should be reevaluated.

Anyway I'm merging this. I'll just ask to look at @lbgracioso comment about PR #51.